### PR TITLE
Add documentation to enable `anbox-cloud` service

### PR DESCRIPTION
**Pull request by keirthana**
Update the appliance topic to replace separate steps of installing AMC, LXD and the appliance snap with steps to enable the `anbox-cloud` service through Ubuntu Pro client.

This change should not go out until September (1.19.1?) when the pro client changes will be rolled out.

### CODE REVIEW

File: tut/installing-appliance.md [hunk 1]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
+    pro enable anbox-cloud
 
-Run the following command to install the `anbox-cloud-appliance` snap, which handles the installation and deployment of the Anbox Cloud Appliance:
+Running this command does the following:
 
-    sudo snap install --classic anbox-cloud-appliance
+1. Install the following tools and dependencies, if they are not installed already:
+    * snapd
+    * Anbox Management Client (AMC)
+    * LXD
```
Comment (ajanon) [inline tut/installing-appliance.md:58]: I feel like this is a bit too short for a tutorial. Or should that be installed automatically from the pro enable command above?

File: tut/installing-appliance.md [hunk 2]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
 
-<a name="additional-tools"></a>
-### 5. Install additional tools
+        [note type="information" status="Important"] The Anbox Cloud Appliance requires LXD >= 5.0 and hence LXD is installed from the `latest/stable` track. If LXD is already installed but the version is older than 5.0, run ` snap refresh --channel=5.0/stable lxd` to update it.[/note]
 
-Run `sudo snap install amc` to install the Anbox Management Client (AMC).
+1. Install `anbox-cloud-appliance` snap from the `latest/stable` track.
```
Comment (ajanon) [inline tut/installing-appliance.md:62]: Same as above, this seems pretty short.

File: tut/installing-appliance.md [hunk 3]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
+1. Configure the `apt` repositories for Anbox Cloud.
```
Comment (ajanon) [inline tut/installing-appliance.md:63]: That doesn't seem right, why do we need this for the appliance?

**Review by ajanon — CHANGES_REQUESTED** [review]

File: tut/installing-appliance.md [hunk 4]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
-
 <a name="attach-ubuntu-pro"></a>
-### 3. Attach your machine to the Ubuntu Pro subscription
+### 2. Attach your machine to the Ubuntu Pro subscription
 
 The Anbox Cloud Appliance requires a valid Ubuntu Pro subscription.
 
 Before installing the appliance, you must attach the machine on which you want to run the Anbox Cloud Appliance to your Ubuntu Pro subscription. To do so, run the following command, replacing *<pro_token>* with your Ubuntu Pro token:
 
     sudo ua attach <pro_token>
```
[the remark this answers is not in the document]
Comment (keirthana) [reply tut/installing-appliance.md:44]: Good catch, thanks!

File: tut/installing-appliance.md (hunk 1 above)
Comment (keirthana) [reply tut/installing-appliance.md:58]: Running the `pro enable` command does all those automatically. I can see that using the imperative verb form can be confusing and hence I have changed it from "Install" to "Installs" which implies the action is being done by the command.

File: tut/installing-appliance.md (hunk 2 above)
Comment (keirthana) [reply tut/installing-appliance.md:62]: Same as above

File: tut/installing-appliance.md (hunk 3 above)
Comment (keirthana) [reply tut/installing-appliance.md:63]: Yes, I added this from the spec but need to check with @morphis if this is being done by the command and also is it really necessary for the users to know.

**Review by ajanon — APPROVED** [review]
LGTM

File: tut/installing-appliance.md [hunk 5]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
 
-    sudo snap install --classic anbox-cloud-appliance
+1. Installs the following tools and dependencies, if they are not installed already:
+    * snapd
+    * Anbox Management Client (AMC)
+    * LXD
 
-<a name="additional-tools"></a>
-### 5. Install additional tools
+        [note type="information" status="Important"] The Anbox Cloud Appliance requires LXD >= 5.0 and hence LXD is installed from the `latest/stable` track. If LXD is already installed but the version is older than 5.0, run `snap refresh --channel=5.0/stable lxd` to update it.[/note]
```
Comment (morphis) [inline tut/installing-appliance.md:60]:
```suggestion
        [note type="information" status="Important"] The Anbox Cloud Appliance requires LXD >= 5.0 and hence LXD is installed from the `5.0/stable` track. If LXD is already installed but the version is older than 5.0, run `snap refresh --channel=5.0/stable lxd` to update it.[/note]
```

Should we also point that if LXD is newer a downgrade is not possible?

File: tut/installing-appliance.md [hunk 6]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
+    pro enable anbox-cloud
 
-Run the following command to install the `anbox-cloud-appliance` snap, which handles the installation and deployment of the Anbox Cloud Appliance:
+Running this command does the following:
 
-    sudo snap install --classic anbox-cloud-appliance
+1. Installs the following tools and dependencies, if they are not installed already:
+    * snapd
+    * Anbox Management Client (AMC)
+    * LXD
```
Comment (morphis) [inline tut/installing-appliance.md:58]: Should we link to the snapcraft.io pages for these?

**Review by morphis — CHANGES_REQUESTED** [review]

File: tut/installing-appliance.md (hunk 3 above)
Comment (morphis) [reply tut/installing-appliance.md]: Yes, we should point this out as discussed in standup. People will see a new APT repository with every `apt update` execution and for that we should explain where it is coming from.

File: tut/installing-appliance.md [hunk 7]
```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
...
 
-    sudo snap install --classic anbox-cloud-appliance
+1. Installs the following tools and dependencies, if they are not installed already:
+    * [snapd](https://snapcraft.io/snapd)
+    * [Anbox Management Client (AMC)](https://snapcraft.io/amc)
+    * [LXD](https://snapcraft.io/lxd)
 
-<a name="additional-tools"></a>
-### 5. Install additional tools
+        [note type="information" status="Important"] The Anbox Cloud Appliance requires LXD >= 5.0 and hence LXD is installed from the `5.0/stable` track. If LXD is already installed but the version is earlier than 5.0, run `snap refresh --channel=5.0/stable lxd` to update it. However, if LXD version is later than 5.0, you cannot downgrade it.[/note]
```
[the remark this answers is not in the document]
Comment (adglkh) [reply tut/installing-appliance.md:60]: Regarding `However, if LXD version is later than 5.0, you cannot downgrade it.`
This sounds a bit confusing to me.  Do you mean people can not downgrade to 5.0 or an earlier version like 4.0? 
Also, the `cannot` is strong emphasis to me. I prefer `do not` over it.

**Review by adglkh — APPROVED** [review]
LGTM

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**